openassessment/workflow/models.py

```diff
@@ -346,12 +355,61 @@ def _get_steps(self):
             # If no steps exist for this AssessmentWorkflow, assume
             # peer -> self for backwards compatibility
             self.steps.add(
-                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=0),
-                AssessmentWorkflowStep(name=self.STATUS.self, order_num=1)
+                AssessmentWorkflowStep(name=self.STATUS.peer, order_num=1),
+                AssessmentWorkflowStep(name=self.STATUS.self, order_num=2)
             )
             steps = list(self.steps.all())
+
+        # A staff step will always be available, to allow for staff overrides
```

Comment:
Oh, I see what you're saying now. I think we're covered there - that's why I set `assessment_completed_at=now()` when we do this, so `is_assessment_complete` will return true. https://github.com/edx/edx-ora2/blob/efischer/staff_endpoint/openassessment/workflow/models.py#L587

Will this cover the situation you're talking about? Do you think we should do the same for `submitted_completed_at`?